.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+sudo: false
+matrix:
+  include:
+  - python: 3.5
+    env: TOXENV=lint
```

Comment:
Counter argument: any Python library written in 2016 should be Python 3 first, 2020 is coming fast.